backup.py

```
@@ -53,16 +54,26 @@
+            return None
+        _docker_clients[host] = client
     return _docker_clients[host]
 
-def set_docker_client(host='local'):
-    if host == 'local':
-        return docker.from_env()
-    else:
-        remote_docker_url = f'tcp://{host}:2375'
-        return docker.DockerClient(base_url=remote_docker_url)
-
+def set_docker_client(host='local', timeout=5):
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)